src/MessagingClient.ts

```diff
@@ -418,10 +417,16 @@ export class MessagingClient implements MessagingAPI {
         await this.matrixClient.setAccountData('m.direct', directRoomMap)
     }
 
-    private getRoomUnreadMessages(room): Array<BasicMessageInfo> {
+    private getRoomUnreadMessages(room: Room | null): Array<BasicMessageInfo> {
+        if (!room) {
```

Comment:
Why would we want to accept this use case?

Reply:
It's not that we want to support it, it's that it's the typing that the functions that use this one have, it would otherwise require a bigger refactor